docs/blog/2025-09-08-some-news.md

```diff
@@ -0,0 +1,59 @@
+
+* The main property checked is implemented in the [eventually-converged.sh](https://github.com/cardano-foundation/antithesis/blob/e86149c883b00b035ff0f61861df7780e7d67e8a/compose/sidecar/composer/convergence/eventually_converged.sh#L1) script and, as its name implies, checks that all nodes have reached the same tip _eventually_. The fact this is an [eventually](https://antithesis.com/docs/test_templates/test_composer_reference/#eventually-command) command is critically important as it guarantees it will be run and its status checked at the end of the test run and without any fault injected. It took us a while to refine the property and work with the AT team to ensure it does not produce false positives,
+* We also check 2 _Never_ properties related to logs emitted by the cardano-node: That it never emits `Error` or `Critical` logs.
+
+Note that by default AT checks a number of other properties, like the fact no container exits unexpectedly, or that the processes run never go over 95% of the memory available.
+
+The team's effort has not been particularly focused on finding bugs, but we still managed to report a couple of potentially "interesting" issues to the core developers: One issue related to DNS resolution failure timeout, and a potential race condition in the shutdown/startup sequence.
+
+## Tracking test runs on-chain
+
+The main goal of this project has always been to open access to the Antithesis resource to the Cardano community, in a transparent, fair, and as easy as possible way. And what's better than a blockchain like Cardano to manage access to a shared resource transparently? Furthermore, as developers of _Cardano_ systems it seems quite logical and inline with the principle of _dogfooding_ that insist on using Cardano itself in our development workflow.
+
+Once convinced about the capabilities of the tool, we started designing and building a Cardano-based command-line application that would allow developers to "easily" request test runs for their particular software stack, and receive results and reports once produced by the platform. Version 0.2.0.0 of this application called _anti-cli_ has [just been released](https://github.com/cardano-foundation/antithesis/releases/tag/v0.2.0.0) and provides the core features needed to manage the lifecycle of AT tests for any system hosted on Github.
+
+_Anti-cli_ leverages [Merkle-Patricia Forestry Service](https://app.radicle.xyz/nodes/seed.hydra.bzh/rad%3AzpZ4szHxvnyVyDiy2acfcVEzxza9) as a backend-service to store data off-chain with proofs verifiable on-chain. This service is in turn based on the excellent Aiken [Merkle-Patricia Forestry](https://github.com/aiken-lang/merkle-patricia-forestry) smart contract.
```

Comment:
There is no such a thing as an MPF smart contract. MPF is offering an aiken lib to assist building smart contracts